pyrealm/pmodel/pmodel_environment.py

```diff
@@ -70,6 +72,13 @@ class PModelEnvironment:
       growing degree days (°C), also used in
       :meth:`~pyrealm.pmodel.quantum_yield.QuantumYieldSandoval`.
 
+    .. note::
```

Comment:
Should there also be a warning output, or is it enough to be here in the docs?

Reply:
I think a warning would be intrusive. I'll need to revise the user docs anyway so will also add a similar note on those pages.